torch/onnx/_internal/fx/passes/type_promotion.py

```diff
@@ -55,6 +67,16 @@ class TypePromotionSnapshot:
     """Expected output dtype of the node."""
 
 
+@_beartype.beartype
+def _get_fake_tensor_val(node: torch.fx.Node) -> fake_tensor.FakeTensor:
+    val = node.meta.get("val", None)
```

Comment:
move inside the if? it makes no sense to set it to none if it will railse on next if

Reply:
Not sure how to do it without accessing node.meta multiple times.
With walrus I can make 1 line shorter but I don't feel it's necessary.